extensions/functions/tensorFlowGeneral/install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
In the etc tflite example models, can we add a license file from the original location?